app/assets/stylesheets/v3/layout_components/side_nav.scss

```diff
@@ -111,6 +112,22 @@
         display: block;
       }
     }
+
+    &:nth-of-type(1),
+    &:nth-of-type(2),
+    &:nth-of-type(3),
+    &:nth-of-type(4) {
```

Comment:
Instead of writing
```
&:nth-of-type(1),
&:nth-of-type(2),
&:nth-of-type(3),
&:nth-of-type(4) {
```

Can we write it like this?
`&:nth-of-type(-n+4) {`